test/AxelarDepositService.js

```diff
@@ -354,5 +357,30 @@ describe('AxelarDepositService', () => {
                 await depositService.connect(ownerWallet).refundLockedAsset(recipient, ADDRESS_ZERO, amount),
             ).to.changeEtherBalance(userWallet, amount);
         });
+
+        it('should have the same receiver bytecode', async () => {
+            const expected = {
+                istanbul: '0xc0fd88839756e97f51ab0395ce8e6164a5f924bd73a3342204340a14ad306fe1',
+                berlin: '0xc0fd88839756e97f51ab0395ce8e6164a5f924bd73a3342204340a14ad306fe1',
+                london: '0xc0fd88839756e97f51ab0395ce8e6164a5f924bd73a3342204340a14ad306fe1',
+            }[EVM_VERSION]
+
+            await expect(keccak256(DepositReceiver.bytecode)).to.be.equal(
+              expected,
+            );
+        });
+
+        it('should have the same proxy bytecode', async () => {
+            const proxyBytecode = DepositServiceProxy.bytecode;
+            const proxyBytecodeHash = keccak256(proxyBytecode);
+
+            const expected = {
```

Comment:
These proxy bytecode doesn't look the same while the name of the test case suggests that they should be?

Reply:
Most of the contracts are compiled differently when targeting a different EVM version. Maybe @re1ro can mention if there's a specific opcode where they differ just to get a better idea. But not really expecting the same one to be generated.

Reply:
That confused me a bit as well, I think changing the test name would help